client/src/containers/Home.jsx

```diff
@@ -1,50 +1,42 @@
 import React from 'react';
 import styles from './Home.module.css';
-import Blog from '../components/Blog';
 import FloatingActionButton from '../components/FloatingActionButton/FloatingActionButton';
+import BlogCard from '../components/BlogCard/index';
+import topTen from '../topTen';
+
+const biggerCards = topTen.slice(0, 2).map((props) => (
```

Comment:
can be use random instead of 0,2 it will be fun, on every reload it will look different

Reply:
The first two will be the top most posts always, sure, we will randomize the other posts to make it more fun! :smile: